runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/StandardIllumina.java

```diff
@@ -71,17 +78,18 @@ public NotificationDto process(File runDirectory, TimeZone tz) throws IOExceptio
     // The Illumina library can't distinguish between a failed run and one that either finished or is still going. Scan the logs, if
     // available to determine if the run failed.
     File rtaLogDir = new File(runDirectory, "/Data/RTALogs");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)